src/httpserver.cpp

```diff
@@ -15,6 +15,7 @@
 #include <sync.h>
 #include <ui_interface.h>
 
+#include <deque>
```

Comment:
Why need to add this include?

Reply:
It's used but not included, probably someone include it indirectly but now it doesn't, so it wont compile
https://github.com/DeFiCh/ain/pull/92/files/10eff033afbd58dd9c77646803b70a6c84d787a5#diff-63c8cb9c9dd61d50d59afd5c39914e1c259f8743030b637a7896a0746c851ef1R76